client/implementation/src/main/java/io/smallrye/graphql/client/dynamic/ResponseImpl.java

```diff
@@ -15,10 +16,12 @@
 
     private final JsonObject data;
     private final List<Error> errors;
+    private List<Map.Entry<String, String>> headers;
```

Comment:
This can be `final` (and a `Map<String, String>`)

Reply:
Changed